clients/lsp-elixir.el

```diff
@@ -176,7 +176,7 @@ be available here: https://github.com/elixir-lsp/elixir-ls/releases/"
    ("elixirLS.mixEnv" lsp-elixir-mix-env)
    ("elixirLS.mixTarget" lsp-elixir-mix-target)
    ("elixirLS.projectDir" lsp-elixir-project-dir)
-   ("elixirLS.fetchDeps" lsp-elixir-fetch-deps t)
+   ("elixirLS.fetchDeps" lsp-elixir-fetch-deps)
```

Comment:
this is not how you do that. This flag marks the property as boolean

Reply:
Sorry, my bad. I fixed by changing `lsp-elixir-fetch-deps` from `t` to `nil` instead. Please kindly review again.